types/parameterTypes.ts

```diff
@@ -28,9 +28,9 @@ export interface Parameter {
   id: string
```

Comment:
Changes to this file are to bring types in line with actual outputs of the R API, which returns 'undefined's (or absences of properties) rather than 'null's.